src/Loop.zig

```diff
@@ -32,6 +33,9 @@ pub fn Loop(comptime T: type) type {
                 .windows => {},
                 else => {
                     if (!builtin.is_test) {
+                        if (self.winch_eventfd < 0) {
+                            self.winch_eventfd = try std.posix.eventfd(0, 0);
```

Comment:
`eventfd` isn't available on macOS or the BSDs. This would need to move a .linux block